ospp/README.md

```diff
@@ -0,0 +1,90 @@
+# About Open Source Promotion Plan
+
+The Open Source Promotion Plan is a summer program organized by the Open Source
+Software Supply Chain Promotion Plan of the Institute of Software Chinese
+Academy of Sciences from 2020. It aims to encourage university students to
+actively participate in the development and maintenance of open source
+software, cultivate and discover more outstanding developers, promote the
+vigorous development of excellent open source software communities, and assist
+in the construction of open source software supply chains.
+
+The program collaborates with various open source communities, providing
+project development tasks for the development and maintenance of important open
+source software, and is open for registration to university students worldwide.
+Students can independently choose projects they are interested in, apply, and
+after selection, carry out development under the guidance of project developers
+(community mentors). Based on the project's difficulty and completion,
+participants (*mentors and mentees*) will receive a bonus, and a completion
+certificate for mentees.
+
+# Who can Mentor
+
+Regular contributors, maintainers and gatekeepers of Rust-VMM can participate
+as mentors. Mentorship involves about 5 hours of time commitment per week to
+communicate with the intern, review their patches, etc. Time is also required
+during the intern selection phase to communicate with applicants. Being a
+mentor is an opportunity to help someone get started in open source
+development, will give you experience with managing a project in a low-stakes
+environment, and a chance to explore interesting technical ideas that you may
+not have time to develop yourself.
+
+Please read mentor guide:
+https://blog-en.summer-ospp.ac.cn/archives/mentor-guide
+
+# How to Participate
+
+As an organization, RustVMM need a `community contact` who is responsible for
```

Comment:
As I mentioned, I can be the `community contact` but I'd open another PR so we can agree on that.